lib/screens/wallet/components/transaction_details_bottom_sheet.dart

```diff
@@ -85,12 +97,14 @@ class TransactionDetailsBottomSheet extends StatelessWidget {
                           Text(transaction.quantity.seedsFormatted, style: Theme.of(context).textTheme.headline5)
                         ],
                       ),
-                      if (transaction.memo.isNullOrEmpty) const SizedBox.shrink() else const SizedBox(height: 20.0),
-                      if (transaction.memo.isNullOrEmpty)
-                        const SizedBox.shrink()
-                      else
-                        Text("Memo: ${transaction.memo}", maxLines: 4, overflow: TextOverflow.ellipsis),
-                      const SizedBox(height: 20.0),
+                      Padding(
+                        padding: const EdgeInsets.symmetric(vertical: 20.0, horizontal: 40.0),
+                        child: Text(
+                          "Memo: ${transaction.memo.isEmpty ? '---' : transaction.memo}",
+                          maxLines: 4,
```

Comment:
I think this bottom-sheet should show the complete memo.

Reply:
??
It shows the full memo, if this is empty will show `Memo:

Reply:
Yes but, maxlines: 4, If is a small device it may not show completely. Let me double check

Reply:
Fixed check new screenshoots, also if you could check the UI for you smaller simulator.

Reply:
![image](https://user-images.githubusercontent.com/51983923/153258582-6f509357-156e-40c4-8736-9b8d0cd2fac0.png)

Bottom overflow is caused by the memo, The right overflow is also in master but the leaves image is the one breaking it.

Reply:
Go it thanks to check, I'll have to install that device (my PC has a clean Android studio).
Maybe I need request to Jason new leaves SVGs :s

Reply:
i can take care of it if you want. is easier for me I have the device already

Reply:
Sounds good to me, I hope we don't need wait for new svgs

Reply:
I did the fix for small devices could you check it.
I record this usign the emulator idk why is so bad and intermittent.
https://user-images.githubusercontent.com/42857405/153330953-649f6b44-e250-493c-81b1-44e2311893ab.mp4

Reply:
is good now, I double check